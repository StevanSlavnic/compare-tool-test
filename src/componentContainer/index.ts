// Container Components
import { ProductsList } from 'componentContainer/ProductsList/ProductsList'
import { ProductsSideBar } from 'componentContainer/ProductsSideBar/ProductsSideBar'
import FeaturesList from 'componentContainer/FeaturesList/FeaturesList'

export { ProductsSideBar, ProductsList, FeaturesList }
