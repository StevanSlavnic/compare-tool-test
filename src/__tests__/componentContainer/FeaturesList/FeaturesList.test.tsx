import React from 'react'
import { shallow } from 'enzyme'
import { FeaturesList } from 'componentContainer'

it('renders FeaturesList', () => {
  const component = shallow(<FeaturesList />)
  expect(component).toMatchSnapshot()
})
