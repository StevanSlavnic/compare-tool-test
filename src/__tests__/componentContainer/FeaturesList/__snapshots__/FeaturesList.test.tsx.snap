// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FeaturesList 1`] = `ShallowWrapper {}`;
